5._Governance.md

```diff
@@ -6,57 +6,64 @@ As a community effort organized within the [Supply Chain Integrity WG](https://g
 * participating in strategic planning, such as coordinating face-to-face meetings or suggesting and approving changes to the governance model
-* creating or restructuring workstreams
+* managing the lifecycle of existing and proposed Workstreams within the Project with the consensus input from the community of Participants
+* creating or restructuring Workstreams
 * responding to specific issues or concerns above and beyond the domain of the various workstreams
 * making decisions when community consensus cannot be reached, pursuant to the appeal process documented below
+* holding Maintainers accountable to the standards of what makes SLSA an effective Project and representative of a diverse community of Participants and Contributors. In the extreme event that a majority of existing Maintainers are unavailable to approve additions or removals for an extended period of time, the Steering Committee may add or remove Maintainers to bring the Project back to a functioning state
+* administration of slsa-framework GitHub organization, including maintaining team memberships for Maintainers, setting security defaults such as branch protection for the organization, managing secrets, etc.
+
 
-**1.5. Steering Committee Member Terms.** During the initial year, the Steering Committee Members will agree on grouping themselves into two groups, one to serve an initial two-year term, and the other for an initial one-year term. Thereafter, the Steering Committee Members will serve two year terms.
+**1.5. Steering Committee Member Terms.** Steering Committee Members have a one year term.
+One month before the expiration of a Steering Committee Member term, any Participant may submit a nomination to fill the seat. A Participant must include the following documentation in their nomination:
 
-At the expiration of a Steering Committee Member term, any Participant may submit a nomination to fill the seat. An individual may be nominated for and serve any number of successive terms.
+* Statement of intent to perform the duties of a Steering Committee Member during the next term
+* Evidence of active participation in the community and being well-versed in SLSA
+* Affiliation / employer
 
-If a Steering Committee Member resigns or ceases to participate for a significant period of time prior to the end of their term, the remaining Steering Committee Members may choose to remove that Steering Committee Member. If so, the remaining Steering Committee Members will determine whether and when to fill the role.
+An individual may be nominated for and serve any number of successive terms. However, Steering Committee Members are encouraged to allow other longstanding Participants to contribute as Steering Committee Members. Steering Committee Members are elected by current Participants. Steering Committee Members must resign if they become inactive. Steering Committee Members may be removed in extraordinary circumstances with an escalation to the OpenSSF TAC.
```

Comment:
> inactive

is this defined formally anywhere?

Reply:
So, we had talked previously around having the steering committee maybe having their own meeting but we found that those meetings don't really address anything that isn't in the spec meeting so I think the idea here should be the same rules as participation with some time limit, e.g. expectations that they are participating in some capacity monthly?

Reply:
If we define inactivity as not performing the duties as stated in 1.4, do you think that is sufficient? To address not being present in meetings, we could add a duty of "active participation in weekly community meetings"? 

I'm hesitant to say monthly or any time period to avoid being able to game this, e.g. show up exactly once a month.

Reply:
Agree: it's hard to be there for all the meetings, and maybe some folks are more active in PRs (which I think matter more) than meetings.

Reply:
How about phrasing this as "active participation with the community"? I recognize that the meeting might not work for all timezones, but I think there are still ways to be an active leader in the community - regular interactions over issues and Slack, leaving comments on the community meeting doc, outreach at conferences, etc.

Activity on PRs can be one component of active participation, but not the only component in my opinion, as that would be more of a Maintainer.

Reply:
As long as the OSSF TAC can use the new phrasing for fair adjudication, sounds reasonable to me.